java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -305,6 +306,22 @@ public T get(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Gets a string value associated with the given <code>key</code> and deletes the key. {@see
+     * #get}, except for the fact that it also deletes the key on success (if and only if the value of
+     * the <code>key</code> is a string).
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/getdel/">redis.io</a> for details.
+     * @param key The <code>key</code> to retrieve from the database.
+     * @return Command Response - If <code>key</code> exists, returns the <code>value</code> of <code>
+     *     key</code> as a <code>String</code>. Otherwise, return <code>null</code>.
```

Comment:
```suggestion
     * @return Command Response - If <code>key</code> exists, returns the <code>value</code> of <code>
     *     key</code>. Otherwise, return <code>null</code>.
```